src/app/global/local_account_sensitive_settings.nim

```diff
@@ -254,56 +243,6 @@ QtObject:
     write = setExpandUsersList
     notify = expandUsersListChanged
 
-
-  proc recentEmojisChanged*(self: LocalAccountSensitiveSettings) {.signal.}
-  proc getRecentEmojis*(self: LocalAccountSensitiveSettings): QVariant {.slot.} =
-    getSettingsPropQVariant(self, LSS_KEY_RECENT_EMOJIS, newQVariant(DEFAULT_RECENT_EMOJIS))
-  proc setRecentEmojis*(self: LocalAccountSensitiveSettings, value: QVariant) {.slot.} =
-    setSettingsProp(self, LSS_KEY_RECENT_EMOJIS, value):
```

Comment:
Could we have used a JSON string instead of a QVariant?

Also, will the old recent emojis be lost because we changed the way it was handled? It's not the end of the world, just annoying

Reply:
Though I do agree that handling the settings in the QML is cleaner, since only the UI cares about those settings

Reply:
Yeah, unless we need that setting also in NIM, this is way cleaner. I also considered going thru JSON but that would only inflate the NIM boilerplate code

Reply:
So the recent emojis were not synced between devices and it's only local thing, right?

Reply:
yes